recipes/unifuncnet/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "unifuncnet" %}
+{% set version = "1.0.2" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/UniFuncNet/archive/refs/tags/{{ version }}.tar.gz
+  sha256: db3a4b803404751fda7d2c82959bca0405593f885292083b1f3db7165278236f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  entry_points:
+    - unifuncnet=unifuncnet.__main__:main
+
+requirements:
+  host:
+    - setuptools >=42
+    - wheel
+    - pip
+    - python >=3.9
+  run:
+    - python >=3.9
+    - requests
+    - beautifulsoup4 
+    - lxml 
```

Comment:
Sorting the list alphabetically makes it easier to read :) 

For example:

```yaml
requirements:
  host:
    - pip
    - python >=3.9
    - setuptools >=42
    - wheel
  run:
    - beautifulsoup4 
    - lxml 
    - python >=3.9
    - requests
```

Reply:
I added your proposed changes